standards/application/waku-api.md

```diff
@@ -107,90 +107,152 @@ language_mappings:
+          static_servers:
             type: array<string>
             default: []
-            description: "Only the passed nodes are used for store queries, discovered store nodes are discarded."
+            description: "Array of server nodes' multiaddresses. Discovered nodes may also be considered."
+            examples:
+              ex1: ["/dns4/node-01.do-ams3.waku.sandbox.status.im/tcp/30303/p2p/16Uiu2HAmNaeL4p3WEYzC9mgXBmBWSgWjPHRvatZTXnp8Jgv3iKsb"]
+              ex2: ["/dns4/node-01.do-ams3.waku.sandbox.status.im/tcp/8000/wss/p2p/16Uiu2HAmNaeL4p3WEYzC9mgXBmBWSgWjPHRvatZTXnp8Jgv3iKsb"]
           clusterId:
             type: uint
+            constraints: mode == "server"
             default: 1
-          sharding_mode:
-            constraints: ["auto", "static"]
-          auto_sharding_config:
-            type: optionAutoShardingConfig
-            default: none
-            description: "The auto-sharding config, if sharding mode is `auto`"
-          active_relay_shards:
+            description: "The cluster subscribed to and participating in. A cluster is composed by multiple shards."
+          shards:
             type: array<uint>
-            constraints: operating_mode == "relay"
-            default: []
-            description: "The shards for relay to subscribe to and participate in." 
-      store_confirmation:
-        type: bool
-        default: false
-        description: "No-payload store hash queries are made to confirm whether outbound messages where received by remote store node."
-          
-  AutoShardingConfig:
-    type: struct
+            constraints: mode == "server"
+            default: [0, 1, 2, 3, 4, 5, 6, 7]
+            description: "The shards subscribed to and participating in."
+  ErrorCode:
+    type: uint
+    constraints: [0 (Ok), 1 (Error), 2 (MissingCallback), 3 (WakuNotResponding)]
+    description:
+      - 0: The function call was successful and finished on time.
+      - 1: Generic error when invoking a particular function. The error detail will be given in the msg callback field.
+      - 2: If, when invoking a certain function, a callback is not set properly.
+      - 3: When the waku node is blocked in a certain callback, or due to other reason, for too long.
```

Comment:
I suggest that for now, errors, their types and values are left to the implementor